arm/CHANGELOG.md

```diff
@@ -21,9 +21,9 @@ updated in this release):
 - *arm/notificationhubs 2014-09-01*
 - arm/redis 2015-08-01
 - *arm/resources/authorization 2015-01-01*
-- *arm/resources/features 2014-08-01-preview*
```

Comment:
@garimakhulbe this is wrong. you should not edit existing changelog item. The part you edited was for v0.4.0, which is already released. (but not tagged, @brendandixon any ideas why? again makes me think we should have a checklist for releases)

anyhow, please create a new section (such as 0.5.0 and add the changes for this version there).